lib/alchemy/solidus/engine.rb

```diff
@@ -44,6 +44,15 @@ class Engine < ::Rails::Engine
           end
         end
       end
+      
+      # In versions of Solidus prior to 2.8, we override the tabs partial
+      # to pass a match_path value to each tab. (Version 2.8 is already
+      # passing this option.) This option is used to configure the paths
+      # for which a given tab is active.
+      #
+      if Spree.solidus_gem_version < Gem::Version.new('2.8')
+        paths['app/views'] << 'lib/match_path'
```

Comment:
I think `lib/views` would be a more recognizable name for this folder. WDYT?

Reply:
I thought it made more sense to have a feature-specific folder name since the folder is being conditionally included in the path. But I'll change it.

Reply:
Done!